src/store/peer/sagas.ts

```diff
@@ -62,10 +74,92 @@ export function* deletePeer(action: ReturnType<typeof actions.deletedPeer.reques
   }
 }
 
+export function* saveGroups(action: ReturnType<typeof actions.saveGroups.request>): Generator {
+  try {
+    yield put(actions.setSavedGroups({
+      loading: true,
+      success: false,
+      failure: false,
+      error: null,
+      data: null
+    }))
```

Comment:
Codacy has a fix for the issue: [Replace `····}))` with `······})⏎····);`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
      })
```